### ISSUE TITLE
California Cases and Deaths 12-16-2020

### ISSUE THREAD

**Issue by DavidViral** [CSSEGISandData/COVID-19#3472]
JH reports Total Cases in state series as 1708559 vs CA Dashboard of 1671081.  JH New Cases shows a huge unrealistic jump in new cases for 12-16 of 63817.  The Covid Tracking Project reports the same numbers as CA.

JH Deaths reported 21817. CA reports 21481.

CA reported total cases are "confirmed", but I see no probable or positive antigen cases reported that might have been added to the confirmed by JH.  Also the CTP would have added probables if CA reported them.  I don't know where the JH numbers come from.  

Additionally, CA posted this note today:
This note has been updated. The previous note included a breakdown of cases reported related to a newly implemented auto processing feature. This was an estimate based on an initial review of the data. Since then, the California Department of Public Health further reviewed and validated the data. As a result, there are a total of 15,337 cases from today's 53,711 positive cases that come from prior days and were processed for reporting today.

I am lost!

**Comment by NaKo91** [comment]
Yesterday's numbers seem to be valid, as Worldometers shows the same jump in cases and deaths.
They have a useful oversight page for almost every state on the county level. Data is from local DPHs, not from the state.  

https://www.worldometers.info/coronavirus/usa/california/

**Comment by DavidViral** [comment]
> 
> 
> Yesterday's numbers seem to be valid, as Worldometers shows the same jump in cases and deaths.
> They have a useful oversight page for almost every state on the county level. Data is from local DPHs, not from the state.
> 
> https://www.worldometers.info/coronavirus/usa/california/

Even using the California numbers and presumably JH numbers you need to back-fill  15337 cases into 12-15 and earlier, thus reducing 12-16 new cases by that same number.  That puts JH new cases for 12-16 at 48480 - which is very high but credible.

**Comment by NaKo91** [comment]
The JHU always attempts to redistribute spiking cases and deaths stemming from backlogs in reporting. To do that, they need a correct timeline from the states, though. If the CA dashboard doesn't reassign backlogged cases within its timeline, the JHU won't do so as well. As far as I know, in such cases, the JHU eventually reaches out to the state for more information but doesn't always get the data they need.

**Comment by DavidViral** [comment]
Yes that's my observation as well..I have had to backfill perhaps a dozen states at times based on a guesstimate of distribution.

Sent from Yahoo Mail on Android 
 
  On Thu, Dec 17, 2020 at 4:55 PM, Nadja<notifications@github.com> wrote:   


The JHU always attempts to redistribute spiking cases and deaths stemming from backlogs in reporting. To do that, they need a correct timeline from the states, though. If the CA dashboard doesn't reassign backlogged cases within its timeline, the JHU won't do so as well. As far as I know, in such cases, the JHU eventually reaches out to the state for more information but doesn't always get the data they need.
